xml/phrases-decl.ent

```diff
@@ -150,6 +150,17 @@
         <para>
          Contains the FIPS certification packages.
         </para>
+        <para>
+         <emphasis>Dependencies:</emphasis> Server Applications
+        </para>
+       </listitem>
+      </varlistentry>
+      <varlistentry os="sles">
+       <term>Confidential Computing Technical Preview</term>
```

Comment:
Should this be "Technology Preview"?

Reply:
That's exactly why I filed https://github.com/SUSE/doc-styleguide/issues/295

In this case it should be "Technical Preview", because this is what the module is actually named in SCC and the YaST installer.